server/meshmodel/aws-rds-controller/relationships/hierarchical_parent.json

```diff
@@ -0,0 +1,51 @@
+{
+    "apiVersion": "core.meshery.io/v1alpha1",
+    "kind": "Hierarchical",
+    "metadata": {
+        "description": "A relationship that defines whether a component can be a parent of other components. It is read like: `from` can be a parent of `to`, where from and to are any components that match the from and to selectors respectively"
+    },
+    "model": {
+        "id": "00000000-0000-0000-0000-000000000000",
+        "name": "aws-rds-controller",
+        "version": "v1.2.4",
+        "displayName": "AWS Relational Database Service",
+        "status": "",
+        "hostID": "00000000-0000-0000-0000-000000000000",
+        "category": {
+         "name": "App Definition and Development",
+         "metadata": null
+        },
+        "metadata": {
+         "isAnnotation": false,
+         "published": true,
+         "subCategory": "Database",
```

Comment:
Remove all of this extraneous information. Follow the Relationships contributor guide. Use the Meshery Integration spreadsheet to create relationship definitions.